frontend/src/components/common/LineChart.vue

```
@@ -22,6 +32,7 @@
 
         data() {
             return {
+                chartDate: this.data,
```

chartDate: this.data  // It seems to me that the error is hidden here, because the data is not received (((

---

get rid of chartDate